boxoffice/views/admin.py

```diff
@@ -35,3 +38,16 @@ def jsonify_org(data):
 @render_with({'text/html': 'index.html', 'application/json': jsonify_org}, json=True)
 def org(organization):
     return dict(org=organization, title=organization.title)
+
+
+@app.route('/admin/o/<org>/revenue')
```

Comment:
Version number missing. Also, why is this under `/admin` and not the agreed on format of `/api/<version>/<resource>/<action>`?